treeherder/model/fixtures/repository.json

```diff
@@ -739,5 +739,30 @@
         "repository_group": 2,
         "description": ""
     }
+},
+{
+    "pk": 61,
+    "model": "model.repository",
+    "fields": {
+        "dvcs_type": "git",
+        "name": "Bugzilla",
+        "url": "https://github.com/bugzilla/bugzilla",
+        "active_status": "active",
+        "codebase": "taskcluster",
```

Comment:
I'd use 'bugzilla' as codebase (for both bugzilla and BMO), since other project's values are things like {gecko, gaia, jetpack, comm}.